lib/PaymentRails/ResourceCollection.php

```diff
@@ -41,9 +41,9 @@ class ResourceCollection implements Iterator
     public function  __construct($response, $items, $pager)
     {
         // Add "meta" fields only when they exist in JSON response
-        $this->_page = isset($response["meta"]) ? $response["meta"]["page"] : null;
+        $this->_page = isset($response["meta"]) ? $response["meta"]["page"] : 1;
         $this->_items = $items;
-        $this->_records = isset($response["meta"]) ? $response["meta"]["records"] : null;
+        $this->_records = isset($response["meta"]) ? $response["meta"]["records"] : count($items);
```

Comment:
Just to clarify that if meta not presented - we fallback to display the count of items of the page instead of the total number of records.

It should be ok since `maximumCount()` which returns $this->_records is not involved with any meaningful calculation